site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -99,250 +111,249 @@ The following table lists the currently available permissions:
     </tr>
     <tr id="declarativeNetRequestFeedback">
       <td><code>"declarativeNetRequestFeedback"</code></td>
-      <td>Grants the extension access to events and methods within the <a
-          href="/docs/extensions/reference/declarativeNetRequest/">chrome.declarativeNetRequest</a> API which return information on declarative
+      <td>Gives access to events and methods within the <a
+          href="/docs/extensions/reference/declarativeNetRequest/">DeclarativeNetRequest</a> API which return information on declarative
         rules matched.</td>
     </tr>
     <tr id="declarativeWebRequest">
       <td><code>"declarativeWebRequest"</code></td>
-      <td>Gives your extension access to the <a href="/docs/extensions/reference/declarativeWebRequest/">chrome.declarativeWebRequest</a> API.</td>
+      <td>Gives access to the <a href="/docs/extensions/reference/declarativeWebRequest/">DeclarativeWebRequest</a> API.</td>
     </tr>
     <!-- No corresponding reference entry
     <tr id="displaySource">
       <td><code>"displaySource"</code></td>
-      <td>Gives your extension access to the <a href="/docs/extensions/reference/displaySource/">chrome.displaySource</a> API.</td>
+      <td>Gives access to the <a href="/docs/extensions/reference/displaySource/">displaySource</a> API.</td>
     </tr>
     <tr id="dns">
       <td><code>"dns"</code></td>
-      <td>Gives your extension access to the <a href="/docs/extensions/reference/dns/">chrome.dns</a> API.</td>
+      <td>Gives access to the <a href="/docs/extensions/reference/dns/">dns</a> API.</td>
     </tr>
     -->
     <tr id="desktopCapture">
       <td><code>"desktopCapture"</code></td>
-      <td>Gives your extension access to the <a href="/docs/extensions/reference/desktopCapture/">chrome.desktopCapture</a> API.</td>
+      <td>Gives access to the <a href="/docs/extensions/reference/desktopCapture/">DesktopCapture</a> API.</td>
     </tr>
     <tr id="documentScan">
       <td><code>"documentScan"</code></td>
-      <td>Gives your extension access to the <a href="/docs/extensions/reference/documentScan/">chrome.documentScan</a> API.</td>
+      <td>Gives access to the <a href="/docs/extensions/reference/documentScan/">DocumentScan</a> API.</td>
     </tr>
     <tr id="downloads">
       <td><code>"downloads"</code></td>
-      <td>Gives your extension access to the <a href="/docs/extensions/reference/downloads/">chrome.downloads</a> API.</td>
+      <td>Gives access to the <a href="/docs/extensions/reference/downloads/">Downloads</a> API.</td>
     </tr>
     <tr id="enterprise.deviceAttributes">
       <td><code>"enterprise.deviceAttributes"</code></td>
-      <td>Gives your extension access to the <a
-          href="/docs/extensions/reference/enterprise_deviceAttributes/">chrome.enterprise.deviceAttributes</a> API.</td>
+      <td>Gives access to the <a
+          href="/docs/extensions/reference/enterprise_deviceAttributes/">Enterprise.deviceAttributes</a> API.</td>
```

Comment:
I'm a bit unsure about this casing and at first glance preferred the old way, since the API namespace is meaningful to developers but `Enterprise.deviceAttributes` (as far as I know) is casing which isn't used elsewhere. Could you say some more on the change? 😀

Reply:
@oliverdunk I wanted to check if the new casing proposal "fit", but I realized this PR is not a great place for it 😶. 
I reverted to the old way for now. We can have a conversation with the team in the future.

Reply:
I'm unsure about this too. In JS, upper case generally indicates a class, which this isn't. I'm also concerned that someone might think they can type 'Enterprise.something' in a piece of code.